src/util/GeoSparqlHelpers.h

```diff
@@ -7,80 +7,29 @@
 
 #include <string>
 
-// TODO: How can I declare the lambdas in the .h file and implement them in the
-// .cpp file? The best I managed so far is define them with inline in the .f
-// file. If I define them without inline in the .cpp file, I get a multiple
-// definition error.
-//
-// namespace ad_utility {
-//
-// // Longitude of a WKT point (it's the first coordinate).
-// std::function<double(const std::string& p)> wktLongitude;
-//
-// // Latitude of a WKT point (it's the second coordinate).
-// std::function<double(const std::string& p)> wktLatitude;
-//
-// // Distance between two WKT points.
-// std::function<double(const std::string& p1, const std::string& p2)> wktDist;
-//
-// }  // namespace ad_utility
+namespace detail {
```

Comment:
I think if your other function live in `ad_utility`, then this namespace can be `ad_utility::detail` (then there also is no need for the `::detail` below.

either
```
namespace ad_utility {
namespace detail {
 // detail stuff
}
// ad_utility stuff
}
```
or 
```
namespace ad_utility::detail { //nested namespaces allowed since c++17
}
namespace ad_utility {
}
`

Reply:
Done!